dev/src/index.ts

```diff
@@ -267,6 +267,13 @@ export class Firestore {
    */
   private _settingsFrozen = false;
 
+  /**
+   * Whether is client has been terminated. Once the Firestore instance is
+   * terminated, terminated cannot be set to false again, and all subsequent
```

Comment:
This is a bit confusing to read. Can we just remove "terminated cannot be set to false again"?